features/mutationevent.yml

```
@@ -0,0 +1,17 @@
+name: Mutation events
+description: Legacy mutation events like `DOMSubtreeModified`, `DOMNodeInserted`, or `DOMNodeRemoved` watch for changes to the DOM and run an event listener callback when DOM changes occur.
```

My feeling has been that we ought to write descriptions in the same style as other features, if possible.

I'd also like to avoid spec language like deprecated, legacy, and obsolete. We have the chance here to help developers by simplifying: please don't use this. 😅

```suggestion
description: Mutation events like `DOMSubtreeModified`, `DOMNodeInserted`, or `DOMNodeRemoved` fire when the DOM changes occur.
```